charts/system/kubelet-csr-approver/templates/_configmap.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* Define the configmap */}}
+{{- define "kubelet-csr-approver.configmap.config" -}}
+enabled: true
+data:
+    {{- if .Values.providerRegex }}
+    PROVIDER_REGEX: {{ .Values.providerRegex }}
+    {{- end }}
+    {{- if .Values.providerIpPrefixes }}
+    PROVIDER_IP_PREFIXES: "{{ join "," .Values.providerIpPrefixes }}"
+    {{- end }}
+    {{- if .Values.maxExpirationSeconds}}
+    MAX_EXPIRATION_SECONDS: {{ .Values.maxExpirationSeconds | quote }}
+    {{- end }}
+    {{- if .Values.bypassDnsResolution}}
+    BYPASS_DNS_RESOLUTION: {{ .Values.bypassDnsResolution | quote }}
+    {{- end }}
+    {{- if .Values.ignoreNonSystemNode}}
+    IGNORE_NON_SYSTEM_NODE: {{ .Values.ignoreNonSystemNode | quote }}
+    {{- end }}
+    {{- if .Values.allowedDnsNames}}
+    ALLOWED_DNS_NAMES: {{ .Values.allowedDnsNames | quote }}
+    {{- end }}
+    {{- if .Values.bypassHostnameCheck}}
+    BYPASS_HOSTNAME_CHECK: {{ .Values.bypassHostnameCheck | quote }}
+    {{- end }}
+    {{- with .Values.env }}
+    {{ toYaml . | nindent 12 }}
+    {{- end }}
```

Comment:
These are just env-vars.
Most have defaults as well, so will essentially always be true

Also, we don't allow .values.env.

Needs to be moved to container config env section.